### PR TITLE
Removed caniusepython3 from api-doc-tools

### PR DESCRIPTION
**Pull request by sksankarraj**
**Description:** Removing caniusepython3 from api-doc-tools

**JIRA:** https://openedx.atlassian.net/browse/BOM-1664

**Merge checklist:**
- [ ] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @sksankarraj! I've created [OSPR-4604](https://openedx.atlassian.net/browse/OSPR-4604) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: .travis.yml [hunk 1]
```diff
@@ -8,15 +8,15 @@ env:
 
 matrix:
   include:
-    # Most IDAs are Python 3.5 and Django 2.2. Test and run codecov on those.
-    - python: 3.5
+    # Most IDAs are Python 3.6 and Django 2.2. Test and run codecov on those.
```
Comment (UsamaSadiq) [inline .travis.yml:11]: Are you sure about this change? I think most of them are still using python35 as we've changed most of the repos to use python35 & python38 and have dropped python36 unless necessary.

Comment (sksankarraj) [reply .travis.yml:11]: I think you are right, but the zipp==3.1.0 requires python3.6, how should I proceed this then?

Comment (mzulqarnain1) [reply .travis.yml:11]: constrain zipp to version <2.0.

Comment (sksankarraj) [reply .travis.yml]: Thanks a lot @mzulqarnain1 , @UsamaSadiq ! I have reverted back to python 3.5 and put those constraints.

**Comment by natabene** [comment]
@sksankarraj Thank you for your contribution! @UsamaSadiq This might be ready for another review. If all looks well to you, can you merge?

File: .travis.yml (hunk 1 above)
Comment (sksankarraj) [reply .travis.yml]: @mzulqarnain1 and @UsamaSadiq , Can you please review this PR again?

**Comment by edx-webhook** [comment]
@sksankarraj 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/api-doc-tools&entry.867055334=https://github.com/edx/api-doc-tools/pull/42&entry.1484655318=https://github.com/sksankarraj&entry.752974735=2020-05-29+03:32&entry.1917517419=2020-06-01+08:03&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**